ui/app/components/app/confirm-page-container/confirm-detail-row/tests/confirm-detail-row.component.test.js

```diff
@@ -1,64 +1,73 @@
 import assert from 'assert'
 import React from 'react'
-import { shallow } from 'enzyme'
 import sinon from 'sinon'
+import configureMockStore from 'redux-mock-store'
+import { fireEvent } from '@testing-library/react'
+import render from '../../../../../../../test/lib/render-helpers'
 import ConfirmDetailRow from '../confirm-detail-row.component'
 
-const propsMethodSpies = {
-  onHeaderClick: sinon.spy(),
-}
-
 describe('Confirm Detail Row Component', function () {
-  describe('render', function () {
-    let wrapper
+  let utils
 
-    beforeEach(function () {
-      wrapper = shallow(
-        <ConfirmDetailRow
-          errorType="mockErrorType"
-          label="mockLabel"
-          showError={false}
-          primaryText="mockFiatText"
-          secondaryText="mockEthText"
-          primaryValueTextColor="mockColor"
-          onHeaderClick={propsMethodSpies.onHeaderClick}
-          headerText="mockHeaderText"
-          headerTextClassName="mockHeaderClass"
-        />,
-      )
-    })
+  const props = {
+    errorType: 'mockErrorType',
+    label: 'mockLabel',
+    showError: false,
+    primaryText: 'mockFiatText',
+    secondaryText: 'mockEthText',
+    primaryValueTextColor: 'mockColor',
+    onHeaderClick: sinon.spy(),
```

Comment:
I realize this is a pre-existing problem with this test suite, but we should avoid creating shared mocks like this. It's quite easy to accidentally not reset spies like this in between tests, and it's also incompatible with the general approach of calling `.restore()` after each test to prevent memory leaks.